packages/contracts/test/voting/token-voting.ts

```diff
@@ -567,17 +591,19 @@ describe('TokenVoting', function () {
       it('does not execute when voting with the `tryEarlyExecution` option', async () => {
         await advanceIntoVoteTime(startDate, endDate);
 
-        await governanceErc20Mock.mock.getPastVotes.returns(50);
-        await voting.vote(id, VoteOption.Yes, false);
+        await voting.connect(signers[0]).vote(id, VoteOption.Yes, false);
+        await voting.connect(signers[1]).vote(id, VoteOption.Yes, false);
+        await voting.connect(signers[2]).vote(id, VoteOption.Yes, false);
+        await voting.connect(signers[3]).vote(id, VoteOption.Yes, false);
+
         expect(await voting.canExecute(id)).to.equal(false);
```

Comment:
could also be optimized with a loop and `Promise.all`

Reply:
Improved in e66e59747b75c6ce535bde30be1342d74a358978